### PR TITLE
UI tests - Unskip upgrade of 1.7.8.0 ~ 1.7.8.2 to the minor and major version on php 7.1 ~ 7.4

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Test fixed issue of disabled module
| Type?             | refactor
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | no
| Sponsor company   | @PrestaShopCorp
| How to test?      | Ci is :green_apple:

### CODE REVIEW

**Review by Progi1984 — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
